### PR TITLE
Setup macOS nightly builds on Travis

### PR DESCRIPTION
**Pull request by nikolaykasyanov**

### CODE REVIEW

**Comment by nikolaykasyanov** [comment]
Now I have to figure out why does Launcher & OpenMW say that "OpenMW 0.41.0 Release" instead of commit SHA.

**Comment by nikolaykasyanov** [comment]
It looks like `GetVersion.cmake` doesn't handle detached HEAD well. Looking into it.

File: components/version/version.cpp [hunk 1]
```diff
@@ -24,7 +24,7 @@ std::string Version::describe()
     std::string str = "OpenMW version " + mVersion;
     std::string rev = mCommitHash;
     std::string tag = mTagHash;
```
Comment (nikolaykasyanov) [inline components/version/version.cpp]: Now it's unused.

**Comment by nikolaykasyanov** [comment]
And now I have to silence Git error messages.

**Comment by nikolaykasyanov** [comment]
All temporary code has been removed, ready for review.

File: .travis.yml [hunk 2]
```diff
@@ -62,6 +64,15 @@ script:
  - if [ "$COVERITY_SCAN_BRANCH" != 1 ] && [ "${TRAVIS_OS_NAME}" = "osx" ]; then make package; fi
  - if [ "$COVERITY_SCAN_BRANCH" != 1 ] && [ "${TRAVIS_OS_NAME}" = "linux" ]; then ./openmw_test_suite; fi
  - if [ "$COVERITY_SCAN_BRANCH" != 1 ] && [ "${TRAVIS_OS_NAME}" = "linux" ]; then cd .. && ./CI/check_tabs.sh; fi
+ - cd "${TRAVIS_BUILD_DIR}"
```
Comment (nikolaykasyanov) [inline .travis.yml]: Travis remembers working dir between `script` and `deploy` steps and can't find deploy script without going back.

**Comment by scrawl** [comment]
That cron job feature is nice. I wonder if we could use that for building the [doxygen page](https://github.com/OpenMW/openmw/tree/doxygen), too.

**Comment by nikolaykasyanov** [comment]
@scrawl it should fit right in.

**Pull request merged**